onnxruntime/core/optimizer/conv_activation_fusion.cc

```diff
@@ -179,6 +179,13 @@ Status ConvActivationFusion::ApplyImpl(Graph& graph, bool& modified, int graph_l
           } else {
             continue;
           }
+        } else if (graph_utils::IsSupportedOptypeVersionAndDomain(next_node, "HardSigmoid", {6})) {
```

Comment:
There are other EPs that also support FusedConv. For a normal ONNX op, this would be a breaking change. How should that be handled?

Reply:
I see. I will limit this fuse within CPU at first. Other EPs could then enhance or modify by theirselves.